internal/database/repository/application.go

```diff
@@ -51,10 +51,10 @@ func applyApplicationFilters(builder *sql.Builder, filters ApplicationFilters) {
 
 func (s *PostgresRepository) UpsertApplications(ctx context.Context, apps []*dao.ApplicationDAOInfo) error {
 	upsertSQL := `INSERT INTO applications (id, app_id, used_resource, max_used_resource, pending_resource,
-			partition, queue_name, submission_time, finished_time, requests, allocations, state,
+			partition, queue_name, queue_id, submission_time, finished_time, requests, allocations, state,
```

Comment:
What's the difference between `queue_id` and the existing `id` column? Do they need to be separate?

Reply:
existing `id` is the id column for the application table, queue_id is `id` column of the `queue` table